bullet_train-outgoing_webhooks/app/models/concerns/webhooks/outgoing/delivery_support.rb

```diff
@@ -7,6 +7,14 @@ module Webhooks::Outgoing::DeliverySupport
 
     has_one :team, through: :endpoint unless BulletTrain::OutgoingWebhooks.parent_class_specified?
     has_many :delivery_attempts, class_name: "Webhooks::Outgoing::DeliveryAttempt", dependent: :destroy, foreign_key: :delivery_id
+
+    after_commit :reset_endpoint_failed_deliveries_marks, if: :delivered?
+  end
+
+  class_methods do
+    def max_attempts_period
+      ATTEMPT_SCHEDULE.values.sum
+    end
```

Comment:
That's only used in the UI now, right? I wonder if we can remove some indirection by doing the UI checks based on the attributes on the endpoint and the `webhooks_outgoing` configs by now? 🤔